Sources/Filters+Strings.swift

```diff
@@ -32,11 +32,39 @@ extension Filters {
       "right", "set", "Type", "unowned", "weak", "willSet"
     ]
 
+    /// Replaces in the given string the given substring with the replacement
+    /// "people picker", replacing "picker" with "life" gives "people life"
+    ///
+    /// - Parameters:
+    ///   - value: the value to be processed
+    ///   - arguments: the arguments to the function; expecting two arguments: substring, replacement
+    /// - Returns: the results string
+    /// - Throws: FilterError.invalidInputType if the value parameter or argunemts aren't string
+    static func replace(_ value: Any?, arguments: [Any?]) throws -> Any? {
+      guard let source = value as? String,
+            arguments.count == 2,
+            let substring = arguments[0] as? String,
+            let replacement = arguments[1] as? String else {
+        throw Filters.Error.invalidInputType
+      }
+      return source.replacingOccurrences(of: substring, with: replacement)
+    }
+
     static func swiftIdentifier(_ value: Any?) throws -> Any? {
       guard let value = value as? String else { throw Filters.Error.invalidInputType }
       return StencilSwiftKit.swiftIdentifier(from: value, replaceWithUnderscores: true)
     }
 
+    /* Lowers the first letter of the string
+     * e.g. "People picker" gives "people picker", "Sports Stats" gives "sports Stats"
```

Comment:
Why the inconsistency of comments style compared to the rest which properly use `///`?